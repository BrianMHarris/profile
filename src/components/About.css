.about-body {
  display: flex;
  justify-content: center;
  width: 100%;
  padding: 60px 0;
  /*background-color: #F4F4F4;*/
  /*background-image: url('../img/background/bg-white2.png')*/
}

.about-content {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  width: 80%;
  max-width: 850px;
  color: var(--clr-txt-main);
}

.about-title {
  font-size: 1.75em;
  text-transform: uppercase;
  font-family: 'Nunito', sans-serif;
  color: var(--clr-txt-standout);
  text-align: center;
}

.about-content p {
  font-size: 1.1em;
  font-family: 'Poppins', sans-serif;
  max-width: 850px;
  margin: 16px 0;
}

@media screen and (min-width: 361px) and (max-width: 767px) {
  .about-content p {
    font-size: 1.25em;
  }

  .about-title {
    font-size: 2.25em;
  }
}

@media screen and (min-width: 768px) {
  .about-content p {
    font-size: 1.5em;
  }

  .about-title {
    font-size: 2.5em;
  }
}

@media screen and (min-width: 1024px) {
  .about-body {
    /*width: 50%;*/
  }
}
