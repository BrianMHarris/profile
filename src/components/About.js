import React from 'react';
import "./About.css"

const About = () => (
  <div className="about-body">
    <div id="about"></div>
    <section className="about-content">
      <div className="about-title">
        about me
      </div>
      <div className="about-flex">
        <p>
          As a kid in the 90's, I got hooked on coding by creating small games and
          apps for AOL.

        </p>
        <p>
          After studying Game Design and Development at Full Sail University,
          I spent 11 years in game design-related roles. I snuck in
          scripting as often I could.
        </p>
        <p>
          As of 2017 I am again following my passion. In web development
          I get to use my skills and imagination to bring amazing things
          to life.
        </p>
      </div>
    </section>
  </div>
)

export default About;
