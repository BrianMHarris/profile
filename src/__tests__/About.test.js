import React from 'react';
import ReactDOM from 'react-dom';
import About from '../components/About';
import renderer from "react-test-renderer";

describe('About', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<About />, div);
  });

  it("matches the snapshot", () => {
    var tree = renderer.create(
        <About />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
