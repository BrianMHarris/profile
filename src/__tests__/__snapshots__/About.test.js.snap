// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About matches the snapshot 1`] = `
<div
  className="about-body"
>
  <div
    id="about"
  />
  <section
    className="about-content"
  >
    <div
      className="about-title"
    >
      about me
    </div>
    <div
      className="about-flex"
    >
      <p>
        As a kid in the 90's, I got hooked on coding through creating little apps for AOL.
      </p>
      <p>
        After studying Game Design and Development at Full Sail University, I spent 11 years in game design-related roles. I snuck in scripting as often I could.
      </p>
      <p>
        As of 2017 I am again following my passion. In web development I get to use my imagination and skills to create and share incredible things.
      </p>
    </div>
  </section>
</div>
`;
